### PR TITLE
split changelog data generation and injection

### PR DESCRIPTION
**Pull request by evgeni**
this allows to re-use the injection method for other things

### CODE REVIEW

File: theforeman.org/pipelines/lib/packaging.groovy [hunk 1]
```diff
@@ -1,3 +1,11 @@
+def debian_release_to_version = [
+    'buster': 'deb10',
+    'bullseye': 'deb11',
+    'bookworm': 'deb12',
```
Comment (evgeni) [inline theforeman.org/pipelines/lib/packaging.groovy:4]: should this say `debian` instead of `deb`?

**Review by ehelms — APPROVED** [review]

File: theforeman.org/pipelines/lib/packaging.groovy (hunk 1 above)
Comment (ehelms) [reply theforeman.org/pipelines/lib/packaging.groovy:4]: That feels cleaner, and if you make that change and want to merge go for it.

Comment (evgeni) [reply theforeman.org/pipelines/lib/packaging.groovy:4]: Will do (tomorrow)

Comment (evgeni) [reply theforeman.org/pipelines/lib/packaging.groovy]: done, and added 13 and 14 codenames as they are already known

**Pull request merged**